.gitignore

```diff
@@ -4,3 +4,4 @@
 .ijwb/
 bazel-*
 .DS_Store
+projectview.bazelproject
```

Comment:
for we dont want it here

Reply:
It was already added in bazel-bsp

Reply:
what do you mean?

Reply:
I believe you added it in this commit: https://github.com/JetBrains/bazel-bsp/commit/e8f205a7b1e3e569b84ccecec60cc5adde2d4604